### PR TITLE
Patch for RelVal Statistics (sumw2) and to run TauValidation on AODSIM

### PR DESCRIPTION
**Pull request by inugent**
Hello,  
  This pull request contains to modifications:
1) the addition of DQMHelper to allow are sumw2 to be used to correct the stats in the RelVal. This should fix the statistics problem in the RelVal comparison plots.
2) Turning off the unused features in the TauValidation to allow for running on AODSim (ie the hepmc collection and weights)
Best regards,
Ian

### CODE REVIEW

**Comment by bendavid** [comment]
+1

**Comment by davidlange6** [comment]
+1

File: Validation/EventGenerator/src/DQMHelper.cc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+DQMHelper::DQMHelper(DQMStore::IBooker *i):ibooker(i){
+
+}
+
+DQMHelper::~DQMHelper(){
+
+}
+
+MonitorElement* DQMHelper::book1dHisto(const std::string &name,const std::string &title,int n,double xmin,double xmax){
```
Comment (deguio) [inline Validation/EventGenerator/src/DQMHelper.cc]: all these couts should be removed asap.
@inugent  please make a PR to fix this.

**Comment by deguio** [comment]
@davidlange6 not sure why the DQM signature is not required here.
is it a mistake or it is like this for historical reasons?

thanks,
F.

**Comment by deguio** [comment]
@inugent the helper classes shouldn't store the ibooker as a private member. however from what I see the ibooker is accessed only within the bookHistograms method so I guess it is fine.

**Comment by bendavid** [comment]
Assuming most validation packages are owned by DQM, no problem to have this one owned by DQM as well.

Is it possible to simply require both dqm and generator signatures here?  (Otherwise we can just watch the package of course) 

Indeed not very familiar with the validation code from our side (but sorry for missing the couts!)

File: Validation/EventGenerator/src/DQMHelper.cc (hunk 1 above)
Comment (inugent) [inline Validation/EventGenerator/src/DQMHelper.cc]: Hello,
     I have update this. So it should be fixed now.
Ian

**Comment by davidlange6** [comment]
I agree its best to co-sign it - @degano , @nclopezo - could you add a DQM signature for Validation/EventGenerator?

**Comment by nclopezo** [comment]
I just added it ( https://github.com/cms-sw/cms-bot/pull/316 )

**Comment by deguio** [comment]
@inugent since this PR has been merged already, another PR is needed.
thanks to all for the quick feedback.
ciao,
F.

**Pull request merged**